snippets/algorithm/distance.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+    Author : Apurva Dubey
+    Date : 06/10/2020
+    Time : 17:17
+    Description : returns the distance of desired position from the first iterator.
+*/
+// demonstrate working of distance() 
+#include <algorithm> 
+#include <iostream> 
+#include <vector> 
+using namespace std; 
+
+int main() 
+{ 
```

Comment:
Braces go in the same line as the starting of the function definition.
Consult Style Guide